packages/fx-core/src/plugins/resource/identity/errors.ts

```diff
@@ -1,11 +0,0 @@
-export class ErrorMessage {
-  public static readonly IdentityLoadFileError = {
-    name: "IdentityLoadFileError",
-    message: () => "Failed to load Azure Resource Manager template files.",
```

Comment:
Since this error is removed, what error will user get if our extension failed to read the bicep template file?